### PR TITLE
[stable/concourse] Add support for encryption key

### PR DESCRIPTION
**Pull request by boazy**
Add support for the new concourse encryption feature described here:
https://concourse.ci/encryption.html

### CODE REVIEW

**Comment by sameersbn** [comment]
@boazy Could you also bump the patch version of the chart.

Also note that the PR cannot be merged unless you sign the CLA https://github.com/kubernetes/charts/pull/1545#issuecomment-317608983

**Comment by boazy** [comment]
Chart version bumped, CLA signed.

File: stable/concourse/templates/web-deployment.yaml [hunk 1]
```diff
@@ -51,6 +51,20 @@ spec:
                   name: {{ template "concourse.fullname" . }}
                   key: postgresql-uri
             {{ end }}
+            {{ if .Values.encryptionKey }}
```
Comment (sameersbn) [inline stable/concourse/templates/web-deployment.yaml:54]: The image tag used in this chart is `2.6.0`. Looking at the https://hub.docker.com/r/concourse/concourse/tags/ page it seems this tag was create 7 months ago. Does this release have support for the encryption feature?

Comment (boazy) [reply stable/concourse/templates/web-deployment.yaml:54]: No. But the image tag can be overridden, and if using version 2.6.0 and no encryption key specified nothing changes.
Perhaps the default tag version should be updated (2.6.0 is outdated by now), but this is another change.

**Pull request merged**